tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
What's a `__backref`? Where does that come from? Google is [iffy](https://www.google.com/search?q=%22__backref%22), as is `ack`:

```
scripts/consistency/ensure_wiki_and_files.py:65:            ('__backrefs',
scripts/consistency/fix_bad_children.py:54:    query = Q('__backrefs.parent.node.nodes.0', 'exists', True)
scripts/consistency/fix_duplicate_addons.py:37:            ('__backrefs',
scripts/dropbox/migrate_to_external_accounts.py:108:                '__backrefs.addons.dropboxusersettings.owner': []
scripts/dropbox/migrate_to_external_accounts.py:121:                '__backrefs.addons.dropboxnodesettings.owner': []
scripts/find_nested_projects.py:14:            Q('__backrefs.parent.node.nodes.0', 'exists', True) &
scripts/fix_templated_orphans.py:54:        Q('__backrefs.parent.node.nodes.0', 'exists', False)
scripts/migrate_github_oauth_settings.py:87:                            '__backrefs.accessed.addongithubusersettings.oauth_settings': raw_user_settings['_id'],
scripts/migrate_github_oauth_settings.py:130:            "__backrefs" : {
scripts/migrate_piwik_base_nodes.py:24:    forked = Q('__backrefs.forked.node.forked_from', 'ne', None)
scripts/migrate_piwik_base_nodes.py:25:    registered = Q('__backrefs.registrations.node.registered_from', 'ne', None)
scripts/migrate_piwik_base_nodes.py:26:    templated = Q('__backrefs.template_node.node.template_node', 'ne', None)
scripts/osfstorage/usage_audit.py:73:    for node in Node.find(Q('__backrefs.parent.node.nodes', 'eq', None)):  # ODM hack to ignore all nodes with parents
website/project/model.py:396:                ('__backrefs.logged.node.logs.$', 1)
website/project/model.py:1851:        query = Q('__backrefs.logged.node.logs', 'in', ids) & Q('should_hide', 'ne', True)
website/util/rubeus.py:191:                Q('__backrefs.parent.node.nodes', 'eq', None)
website/util/rubeus.py:200:                Q('__backrefs.parent.node.nodes', 'nin', all_my_projects.get_keys()) &
website/util/rubeus.py:221:                Q('__backrefs.parent.node.nodes', 'eq', None)
website/util/rubeus.py:230:                Q('__backrefs.parent.node.nodes', 'nin', all_my_registrations.get_keys()) &
```